code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -68,12 +68,18 @@
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/laser
 	equip_cooldown = 8
-	name = "CH-PS \"Immolator\" Laser"
+	name = "CH-PS \"Old Faithful\" Laser"

```

Comment:
@Fox-McCloud Changed to "Firedart". Since that was the only thing you commented on, and it is now changed, merge?